website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,184 @@
+## Structure
+
+A filtering pattern consists of a number of Helios components that work together functionally and enforce a consistent layout. These elements are variable depending on the data set's content and the context within the application. They can be combined in different ways to achieve an experience that benefits the end user.
+
+### Data set
+
+A data set is a broad term for an array of items, objects, or related information presented as separate but related records. A data set is commonly represented via a [Table](/components/table) but may be expressed in other formats depending on the type of information.
+
+**Data set represented by a table**
+
+![Tabular data set](/assets/patterns/filter-patterns/example-data-set-tabular-data.png)
+
+**Data set represented by a grid**
+
+![Non-tabular data set](/assets/patterns/filter-patterns/example-data-set-non-tabular-data.png)
+
+### Filter positioning
+
+Filters are responsible for the functional aspect of limiting a data set and for displaying the [parameters](/patterns/filter-patterns?tab=core%20concepts#parameter) which a user can filter upon. Filter positioning is largely dependent on the complexity of the data set and the scope to which the filters are being applied. For example, are the filters applied at the page level or only to the data set they are paired with?
+
+Filters often consist of numerous parameters that mirror the columns of a table or parameters contained within the data set. For more details on mirroring filters and parameters within a data set, refer to the filtering [core concepts](/patterns/filter-patterns/tab=core%20concepts#mirror-of-parameters).
+
+#### Filter bar
+
+Organizing filters in a **filter bar** is a common positioning method and consists of one or more dropdowns, buttons, or input components that are used to select values from a set of parameters. A [Segmented Group](/components/segmented-group) can be used within a filter bar to group similar parameters and support complex filtering.
+
+A filter bar should be positioned directly above the data set with a a `16px` gap in between.
+
+![Filter bar example](/assets/patterns/filter-patterns/filter-bar-segmented-group.png =728x*)
```

Comment:
(suggestion) you'll want to think about the alt attribute text. The guidelines indicate that the alt attribute value should convey what the image conveys- no more, no less.

Reply:
Yep, good point I'll do a scrub of all of the images